core/src/main/java/com/trendyol/design/core/checkbox/CheckBox.kt

```diff
@@ -182,3 +185,35 @@ private fun TrendyolXSmallCheckboxPreviewDisabled() {
         )
     }
 }
+
+@Preview(showBackground = true)
+@Composable
+private fun TrendyolMediumCheckboxWithTextRTLPreview() {
+    PreviewTheme {
+        CompositionLocalProvider(LocalLayoutDirection provides LayoutDirection.Rtl) {
```

Comment:
Maybe we could create a `PreviewRTLTheme` or add a parameter to `PreviewTheme` named `isRTL`